bioblend/galaxy/forms/__init__.py

```diff
@@ -1,16 +1,27 @@
 """
 Contains possible interactions with the Galaxy Forms
 """
+from typing import (
+    Any,
+    Dict,
+    List,
+    TYPE_CHECKING,
+    Union,
+)
+
 from bioblend.galaxy.client import Client
 
+if TYPE_CHECKING:
+    from bioblend.galaxy import GalaxyInstance
+
 
 class FormsClient(Client):
     module = "forms"
 
-    def __init__(self, galaxy_instance):
+    def __init__(self, galaxy_instance: "GalaxyInstance") -> None:
         super().__init__(galaxy_instance)
 
-    def get_forms(self):
+    def get_forms(self) -> List[Dict[str, str]]:
```

Comment:
Given that the dict returned by Galaxy may change in the future, I'd keep this a bit more general:

```suggestion
    def get_forms(self) -> List[Dict[str, Any]]:
```